hnn_core/tests/test_optimization.py

```diff
@@ -136,3 +136,91 @@ def test_optimize_evoked():
 
     # static drive params should remain constant
     assert drive_static_params_opt == drive_static_params_orig
+
+
+def test_optimize_evoked_selectedDrive():
+    """Test running optimzing a selected drive while parameters of other drives
+    stay the same in a reduced network."""
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+
+    tstop = 10.
+    n_trials = 1
+
+    # simulate a dipole to establish ground-truth drive parameters
+    mu_orig = 6.
+    params.update({'N_pyr_x': 3,
+                   'N_pyr_y': 3,
+                   't_evprox_1': mu_orig,
+                   'sigma_t_evprox_1': 2.,
+                   't_evdist_1': mu_orig + 2,
+                   'sigma_t_evdist_1': 2.})
```

Comment:
there seems to be quite a bit of code duplication with `test_optimize_evoked`. If we add these tests to that function instead of creating a new function, could we reduce this duplication?